### PR TITLE
add build timer

### PR DESCRIPTION
**Pull request by Gireen**
required for buildqueue

### CODE REVIEW

**Comment by bmorel** [comment]
This is a PR to go with the build queue in unvanquished repo, I assume?

**Comment by Gireen** [comment]
Correct. it replaces the progress bar with a text and timer how much additional build time is added.

File: ui/hud_builder.rml [hunk 1]
```diff
@@ -43,22 +44,29 @@ progressbar
...
+text-align:center;
+}
 
 .warning {
 	color: yellow;
 }
 
 .error {
 	color: red;
-}
```
Comment (bmorel) [inline ui/hud_builder.rml:53]: where is the '}' gone?

File: ui/hud_builder.rml [hunk 2]
```diff
@@ -43,22 +44,29 @@ progressbar
 	image: /gfx/colors/white;
 	orientation: left;
 }
+#buildQueueTime
```
Comment (bmorel) [inline ui/hud_builder.rml:47]: looks like there's an empty line for nothing and content is not indented

**Pull request closed without merging**